app/main.py

```diff
@@ -2,24 +2,70 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(
+            self,
+            owner: any,
+            name: str
+    ) -> None:
+        self.protected_name = f"_{name}"
+
+    def __get__(self, instance: any, owner: any) -> str:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: any, value: int) -> None:
```

Comment:
It is better to use `Any` from `typing` since it is not deprecated. Change in all places